sdk/synapse/azartifacts/zz_generated_connection.go

```diff
@@ -0,0 +1,77 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package azartifacts
+
+import (
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+const scope = "https://dev.azuresynapse.net/.default"
```

Comment:
Out of curiosity, is this constant generated from the `credential-scope` parameter of the autorest command?

Reply:
Yes however this will be changing after updating to the latest M4.